card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -37,7 +37,7 @@ def default_modal_content_args args
   end
 
   view :modal_body do |args|
-    _render_content(args)
+    _render_core(args)
```

Comment:
He tried it but it didn't work.  The difference is that the alias attaches the modal_body view to a specific core view at load time, whereas the long definition attaches to it at render time.  We should probably get clear (elsewhere) on whether that's what we want, but that's what we got!